server/schedulers/hot_region.go

```diff
@@ -1079,12 +1095,12 @@ func (bs *balanceSolver) buildOperator() (op *operator.Operator, infl *Influence
 	}
 
 	dim := ""
-	if bs.byteIsBetter && bs.keyIsBetter {
+	if bs.firstPriorityIsBetter && bs.secondPriorityIsBetter {
 		dim = "both"
```

Comment:
```suggestion
		dim = "all"
```

I think `all` is better